src/onnx/parse_conv_transpose.cpp

```diff
@@ -80,14 +78,21 @@ struct parse_deconvolution : op_parser<parse_deconvolution>
                                std::back_inserter(values["padding"]),
```

Comment:
We could copy all the padding values instead of copying untill `pad_ndims`. This could create bugs otherwise.

Reply:
Current `convolution_backwards` op assumes only `ndim` padding values rather than there possibly being `2*ndim`. Changing the `check_attribute_size()` to reflect this.

Reply:
I find it would be better to modify and make `convolution_backwards` take padding values `2 * ndim` so that only during parsing we take care of ASym v/s/ Sym but inside MIGraphX we handle it generally and not rely on assumptions. We could do it as separate PR.